shared/src/cell_set.rs

```diff
@@ -124,69 +132,67 @@ impl CellSet {
         self.inner.get(h)
     }
 
-    pub fn insert(
+    pub(crate) fn put(&mut self, tx_hash: H256, tx_meta: TransactionMeta) {
+        self.inner.insert(tx_hash, tx_meta);
+    }
+
+    pub(crate) fn insert_cell(
+        &mut self,
+        cell: &CellOutPoint,
+        number: u64,
+        epoch: u64,
+        cellbase: bool,
+        outputs_len: usize,
+    ) -> TransactionMeta {
+        let mut meta = if cellbase {
```

Comment:
what if TransactionMeta is already existed?

Reply:
- If we care this, then we should use `insert_transaction(...)`.

- In our scene, the check has already did before we call this function (I call it after `try_mark_live(..)`).